apps/wasm/mvp/readme.md

```diff
@@ -0,0 +1,75 @@
+# Prerequisites
```

Comment:
It would be cool to have an overall description of the architecture of this demo and a high level of how it works

Reply:
That's good suggestion, a UML diagram added 
![arch-uml](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/hanabi1224/mina-rs/wasm-mvp/docs/mvp-arch.iuml)

Reply:
This is really helpful